iceberg/bodo_iceberg_connector/iceberg-java/src/main/java/com/bodo/iceberg/catalog/CatalogCreator.java

```diff
@@ -83,24 +79,14 @@ public static Catalog create(String connStr, String catalogType, String coreSite
       case "hadoop-abfs":
         catalog = HadoopBuilder.create(uriBuilder.removeQuery().build().toString(), conf, params);
         break;
-      case "snowflake":
-        catalog = SnowflakeBuilder.create(conf, params);
-        break;
       case "rest":
         catalog = RESTBuilder.create(conf, params);
         break;
       default:
         throw new UnsupportedOperationException("Should never occur. Captured in Python");
     }
 
-    // CachingCatalog is a simple map between table names and their `Table` object
-    // Does not modify how the Table object works in any way
-    // Can be invalidated by CREATE / REPLACE ops or a timer (if time passed in as
-    // arg)
-    // Benefits: Potentially some speed up in collecting repeated metadata like
-    // schema
-    // Downsides: Does not refresh if another program modifies the Catalog
-    return CachingCatalog.wrap(catalog);
```

Comment:
Removed the CachingCatalog change because it was causing issues when jumping between PyIceberg and Iceberg-Java. This shouldn't be as much as a problem anymore because we use some caching on the PyIceberg side, which is still the predominant caller for getting metadata. Iceberg Java is only used for more heavy handed stuff like commits.